ddln_imaginator.proto

```diff
@@ -1,13 +1,18 @@
 syntax = "proto2";
 
-package ddln;
+package ddln.imaginator;
 
 import "ddln_common.proto";
 
 message Intrinsics {
     required float verticalFieldOfView_deg = 1;
 }
 
+message Object {
+    optional string identifier = 1;
```

Comment:
`identifier` is misleading, it is `type` or similar word
e.g. we can have two separate B747 in the scene

Reply:
It is a vector not a map so there can be two B747 using this notation